drivers/trim_util.py

```diff
@@ -104,15 +104,26 @@ def do_trim(session, args):
         time.sleep(LOCK_RETRY_INTERVAL)
 
     if got_lock:
-        try:
-            vg_name = _vg_by_sr_uuid(sr_uuid)
-            lv_name = sr_uuid + TRIM_LV_TAG
-            lv_path = _lvpath_by_vg_lv_name(vg_name, lv_name)
-
-            # Clean trim LV in case the previous trim attemp failed
-            if lvutil.exists(lv_path):
-                lvutil.remove(lv_path)
```

Comment:
Here the reason i moved few lines of code above try block is that, the methods inside
those functions already have exception handling. 

But as you mentioned the reason for these lines inside try block is because we have to give the appropriate TRIM functionality error to end user.

So i moved the code back.